This has to be patched manually.
In theory we could use config.toml to override this, but Cargo.lock does not have the information
on the path after root.  It could be root, but in this case it's "monoio" and there's no way to
know without opening the source up and inspecting it manually.

In there we could add shell script to do this patch in the post-patch or pre-configure target,
but since monoio will be leaving with version 3, don't bother with it.

--- Cargo.toml.orig	2026-06-17 13:03:47 UTC
+++ Cargo.toml
@@ -24,7 +24,7 @@ ferron-modules-builtin = { path = "./fer
 ferron-observability-builtin = { path = "./ferron-observability-builtin", default-features = false }
 
 [patch.crates-io]
-monoio = { git = "https://github.com/DorianNiemiecSVRJS/monoio.git" }                                           # This is needed for the web server to be able to accept more than one connection on Windows, and to be able to be compiled correctly on FreeBSD
+monoio = { path = "/construction/ferron/monoio-7955c1356585d7ebd45d67f5932620c09ac0a701/monoio" }
 
 [patch.'https://github.com/ferronweb/ferron']
 ferron-common = { path = "./ferron-common" }
